addon/templates/components/bootstrap-selectize.hbs

```diff
@@ -27,7 +27,8 @@
           update-filter=updateFilter
           select-item=selectItem
           add-item=addItem
-          remove-item=removeItem}}
+          remove-item=removeItem
+          tabindex=tabindex}}
```

Comment:
Doesn't look like this is supported: https://github.com/miguelcobain/ember-cli-selectize#properties or https://github.com/brianreavis/selectize.js/blob/master/docs/usage.md#options